web-test-runner.config.mjs

```diff
@@ -0,0 +1,138 @@
+import { importMapsPlugin } from '@web/dev-server-import-maps';
+import { legacyPlugin } from '@web/dev-server-legacy';
+import { createSauceLabsLauncher } from '@web/test-runner-saucelabs';
+import rollupBabel from '@rollup/plugin-babel';
+import { fromRollup } from '@web/dev-server-rollup';
+import fs from 'fs';
+import os from 'os';
+
+let browsers;
+const sauceLabsUser = process.env.SAUCE_USERNAME;
+const sauceLabsKey = process.env.SAUCE_ACCESS_KEY;
+const sauceLabs = sauceLabsUser && sauceLabsKey;
+if (sauceLabs) {
+	const sauceLabsCapabilities = {
+		name: 'unit-tests',
+		// if you are running tests in a CI, the build id might be available as an
+		// environment variable. this is useful for identifying test runs
+		// this is for example the name for github actions
+		build: `CI #${process.env.GITHUB_RUN_NUMBER} (${process.env.GITHUB_RUN_ID})`
```

Comment:
I think you could replace `CI` with `${process.env.GITHUB_WORKFLOW}` ([documentation](https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables))